docs/CONTRIBUTING.md

```diff
@@ -14,17 +14,21 @@ for more information.  Instead, we use an instance of
 
 As a review proceeds, the preferred method is to push `fixup!`
 commits via `git commit --fixup`.  Homu knows how to use
-`--autosquash` when performing the final merge.  See the
+`--autosquash` when performing the final merge.
+
+Any commits committed with the `--fixup` option will have have the word `fixup!` in its commit title. This is to indicate that this particular commit will be squashed (by Homu when batch merging) with the commit that was specified in the command, `git commit --fixup <commit ref or hash>`.
```

Comment:
Minor: note that "batch merging" refers to something else -- it's when Homu combines multiple PRs into a single test to save time -- so that should just say e.g. "by Homu when merging"?